internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
+
+func LoadConfiguration() (Configuration, error) {
```

Comment:
It might be a good idea to validate the configuration after parsing the environment variables, to ensure they meet your application requirements beyond just being non-empty. This will help catch issues early in the load process if additional constraints are necessary.